src/panel/settings/settings_wm.vala

```diff
@@ -80,8 +80,8 @@ namespace Budgie {
 			/* Unredirect.. */
 			switch_unredirect = new Gtk.Switch();
 			grid.add_row(new SettingsRow(switch_unredirect,
-				_("Disable unredirection of windows"),
-				_("This option is for advanced users. Use this if you are having graphical or performance issues with dedicated GPUs.")
+				_("Enable unredirection"),
+				_("Enable unredirection which will allow frames to bypass compositing for fullscreen application. This option is for advanced users and recommended to keep enabled. Use this if you are having graphical or performance issues with dedicated GPUs.")
```

Comment:
*for fullscreen applications

Reply:
Fixed in 79d9f2ab34e17015d55e41fe3afddcf84cfc4335